src/AudioUnit/AUEnums.cs

```diff
@@ -200,6 +202,7 @@ public enum AudioObjectPropertySelector : uint
 		ActualSampleRate = 1634955892,// 'asrt',
 		ClockDevice = 1634755428, // 'apcd',
 		IOThreadOSWorkgroup = 1869838183, // 'oswg'
+		ProcessMute = 1634758765, // 'appm'
```

Comment:
^ needs availability attributes - for both `NET` and legacy

Reply:
if it's not available in tvOS then it needs `[NoTV]` and `[SupportedOSPlatform ("tvos")]`
unless the enum type itself is not available ?

otherwise it implies availability for tvOS 9 (for legacy) or tvOS 10 (for net6)